### PR TITLE
WHIMSY-314: Replaced a.o/img/ with a.o/logos/

### PR DESCRIPTION
**Pull request by rlenferink**
As requested per [WHIMSY-314](https://issues.apache.org/jira/browse/WHIMSY-314) apache.org/img/ has been replaced with apache.org/logos/

### CODE REVIEW

File: www/roster/models/committee.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def self.serialize(id, env)
     info = JSON.parse(File.read(File.join(comdev, 'projects.json')))[id]
 
     image_dir = ASF::SVN.find('site-img')
-    image = Dir[File.join(image_dir, "#{id}.*")].map {|path| File.basename(path)}.last
+    image = Dir[File.join(image_dir, "#{id}*.{svg,eps,ai,pdf}")].map {|path| File.basename(path)}.first
```
Comment (ShaneCurcuru) [inline www/roster/models/committee.rb]: Why the change to .first instead of .last?

File: tools/site-scan.rb [hunk 2]
```diff
@@ -146,7 +146,7 @@ def parse(id, site, name)
   end
   # THIRD: see if an image has been uploaded
   if IMAGE_DIR
-    data[:image] = Dir[File.join(IMAGE_DIR, "#{id}.*")].
+    data[:image] = Dir[File.join(IMAGE_DIR, "#{id}*.{svg,eps,ai,pdf}")].
```
Comment (ShaneCurcuru) [inline tools/site-scan.rb]: Is there a documented reason to scan for only those file types?  Why not include .png as a catch-all (Which is all some projects used to have).

Comment (rlenferink) [reply tools/site-scan.rb]: The reason why I added this, is because [logos/about.html](http://www.apache.org/logos/about.html) states that `svg`, `eps`, `ai` and `pdf` file types are accepted. Checking for PNGs would result in 'Yeah, the logo is present!', however the logo won't be propagated to a.o/logos which IMO can confuse some people as well.

Side note: a lot of originals are already scalable types (check [here](https://svn.apache.org/repos/asf/comdev/project-logos/originals/))

File: www/roster/models/committee.rb (hunk 1 above)
Comment (rlenferink) [reply www/roster/models/committee.rb]: [logos/about.html](http://www.apache.org/logos/about.html) states: `The first logo, lexicographically, will be picked as the default.` Changing this to `.first` will make sure that Whimsy uses the default logo as well.

**Comment by rlenferink** [comment]
> LGTM, modulo questions below. Did you confirm that letting site-scan run gave the same number of pass/fails as before?

@ShaneCurcuru I tested this by executing the following from the Whimsy root and comparing the output:

Implementation using a.o/img/
```bash
[rlenferink@centos8 whimsy]$ ./tools/site-scan.rb https://celix.apache.org celix celix-scan.json
celix https://celix.apache.org recent
{
}
[rlenferink@centos8 whimsy]$ cat celix-scan.json
{
  "celix": {
    "foundation": "ASF Home",
    "events": "https://www.apache.org/events/current-event-234x60.png",
    "license": "https://www.apache.org/licenses/",
    "thanks": "https://www.apache.org/foundation/thanks.html",
    "security": "https://www.apache.org/security/",
    "sponsorship": "https://www.apache.org/foundation/sponsorship.html",
    "trademarks": "Apache Celix, Celix, Apache, the Apache feather logo and the Apache Celix logo are trademarks of The Apache Software Foundation.",
    "copyright": "Copyright © 2020 The Apache Software Foundation",
    "image": "celix.png",
    "display_name": "celix",
    "uri": "https://celix.apache.org"
  }
}
```

Implementation using a.o/logos/
```bash
[rlenferink@centos8 whimsy]$ ./tools/site-scan.rb https://celix.apache.org celix celix-scan.json
celix https://celix.apache.org recent
{
}
[rlenferink@centos8 whimsy]$ cat celix-scan.json
{
  "celix": {
    "foundation": "ASF Home",
    "events": "https://www.apache.org/events/current-event-234x60.png",
    "license": "https://www.apache.org/licenses/",
    "thanks": "https://www.apache.org/foundation/thanks.html",
    "security": "https://www.apache.org/security/",
    "sponsorship": "https://www.apache.org/foundation/sponsorship.html",
    "trademarks": "Apache Celix, Celix, Apache, the Apache feather logo and the Apache Celix logo are trademarks of The Apache Software Foundation.",
    "copyright": "Copyright © 2020 The Apache Software Foundation",
    "image": "celix.svg",
    "display_name": "celix",
    "uri": "https://celix.apache.org"
  }
}
```

By comparing the 'image' item I found that it correctly reads in the image (celix.png -> celix.svg).

Is this test sufficient or do I need to test it in a different way as well?

**Comment by rlenferink** [comment]
@ShaneCurcuru any further thoughts on this?

**Comment by ShaneCurcuru** [comment]
Thanks for the replies, sorry for the delay!
